### DISCUSSION TITLE
"Installing Windows Server" Prompt?

### DISCUSSION THREAD

**Discussion by Mdelapaz2022** [AveYo/MediaCreationTool.bat#136] [General]
I am upgrading my friend's PC (I myself am on a M1 Max MacBook Pro) and it says "Installing Windows Server". I am not a pro at Windows since I mainly use macOS 12.3, Is this normal?

**Comment by AveYo** [comment]
It is normal for the **Skip TPM on Dynamic Update** script - it's mostly cosmetic anyway
Dev (and now Beta too) need such reliable bypass when upgrading via Windows Update. 

For 11 Release, or for upgrading via iso, can still get away with a simpler bypass (`cd.>appraiserres.dll`)
